Utils.lua

```diff
@@ -615,6 +615,23 @@ function UTILS.LibStModifierFnNumber(a1, b1)
     return (tonumber(RemoveColorCode(a1)) or 0), (tonumber(RemoveColorCode(b1)) or 0)
 end
 
+function UTILS.LibStModifierFnVersion(a1, b1)
+    local VersionStringToNumber = function(s)
```

Comment:
This should be abstracted from:
https://github.com/ClassicLootManager/ClassicLootManager/blob/master/ClassicLootManager.lua#L57
And reused in both places.

Comparison should be:
```lua
if major1 > major2 then
    return true
else
    if minor1 > minor2 then
        return true
    else
        if patch1 > patch2 then
            return true
        end
    end
end
return false

```